docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,575 @@
+- Enable test automation and CI/CD, letting your pipelines make use of the project configuration.
+
+## Example configurations
+
+<!-- TODO
+Shall we provide a handful of examples here of different use cases and the .json for each?
+
+Mike B - "I'd cover at least two. One for accessing multiple services directly on multiple LPARs that share the same username/password. One for accessing multiple services via the API ML (where MFA/SSO is achievable via token-based auth).
+
+Another example could be accessing multiple services directly on LPAR1 and LPAR2 where username/password varies between LPAR1 & LPAR2 services.
+
+Yet another example could be a situation where API ML is leveraged to access production services but services on a dev-test environment can be accessed directly."
+
+t1m0thyj - Added JSON examples below
+-->
+```json
+// accessing multiple services directly on multiple LPARs that share the same username/password
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "lpar1": {
+            "properties": {
+                "host": "example1.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 443
+                    }
+                },
+                "tso": {
+                    "type": "tso",
+                    "properties": {
+                        "account": "ACCT#",
+                        "codePage": "1047",
+                        "logonProcedure": "IZUFPROC"
+                    }
+                },
+                "ssh": {
+                    "type": "ssh",
+                    "properties": {
+                        "port": 22
+                    }
+                }
+            }
+        },
+        "lpar2": {
+            "properties": {
+                "host": "example2.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 1443
+                    }
+                }
+            }
+        },
+        "my_base": {
+            "type": "base",
+            "properties": {
+                "rejectUnauthorized": true
+            }
+        }
+    },
+    "defaults": {
+        "zosmf": "lpar2.zosmf",
+        "tso": "lpar1.tso",
+        "ssh": "lpar1.ssh",
+        "base": "my_base"
+    },
+    "plugins": [],
+    "secure": [
+        "profiles.my_base.properties.user",
+        "profiles.my_base.properties.password"
+    ]
+}
+```
+
+```json
+// accessing multiple services via the API ML (where MFA/SSO is achievable via token-based auth)
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "lpar1": {
```

Comment:
@t1m0thyj @MichaelABroadcom for this example, I would recommend not nesting under "lpar1".